src/pages/CardLayout/Subscribe.vue

```diff
@@ -53,10 +50,18 @@ import Input from '@hackthe6ix/vue-ui/Input';
 import {Card} from '@components';
 import {validate, query} from '@utils';
 import {SUBSCRIBE} from '@graphql';
+import formProvider from '@hackthe6ix/vue-ui/utils/mixins/formProvider';
 
 export default {
   name: 'Subscribe',
   path: '/subscribe',
+  inject: ['form_data', 'form_errors'],
```

Comment:
u cant inject from the provider, they are the ones supplying the values for inject XD